src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,205 @@
+            -k <storage-account-key>
+
+Delete a container registry
+-------------
+::
+
+    Command
+        az acr delete: Delete a container registry.
+
+    Arguments
+        --name -n [Required]: The primary resource name.
+
+    Global Arguments
+        --debug             : Increase logging verbosity to show all debug logs.
+        --help -h           : Show this help message and exit.
+        --output -o         : Output format.  Allowed values: json, jsonc, list, table, tsv.  Default:
+                            json.
+        --query             : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        Delete a container registry
+            az acr delete -n <registry-name>
+
+Get a container registry
+-------------
+::
+
+    Command
+        az acr show: Get a container registry.
+
+    Arguments
+        --name -n [Required]: The primary resource name.
+
+    Global Arguments
+        --debug             : Increase logging verbosity to show all debug logs.
+        --help -h           : Show this help message and exit.
+        --output -o         : Output format.  Allowed values: json, jsonc, list, table, tsv.  Default:
+                            json.
+        --query             : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        Get a container registry and show result in a table
+            az acr show -n <registry-name> -o table
+
+Update a container registry
+-------------
+::
+
+    Command
+        az acr update: Update a container registry.
+
+    Arguments
+        --name -n [Required]: The primary resource name.
+        --tags              : Multiple semicolon separated tags in 'key[=value]' format.  Use "" to
+                            clear existing tags.
+
+    Global Arguments
+        --debug             : Increase logging verbosity to show all debug logs.
+        --help -h           : Show this help message and exit.
+        --output -o         : Output format.  Allowed values: json, jsonc, list, table, tsv.  Default:
+                            json.
+        --query             : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        Update tags of a container registry and show result in a table
+            az acr update -n <registry-name> --tags key1=value1;key2=value2 -o table
+
+The catalog of repositories in the specified registry
+-------------
+::
+
+    Command
+        az acr catalog: The catalog of repositories in the specified registry.
```

Comment:
>The catalog of repositories in the specified registry. [](start = 24, length = 54)

This doesn't say what it actually does. Does it return a list? Also, the command name should have a verb, so it would be something like 'list-catalog' or 'show-catalog'.